constants/userStatus.js

```diff
@@ -0,0 +1,7 @@
+const userState = {
```

Comment:
we decided to call the collection status hence its acceptable values as state. and since these are states of user Status hence calling it user state.